maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/impl/FinancialAssessmentImpl.java

```diff
@@ -73,9 +73,8 @@ public FinancialAssessmentEntity create(FinancialAssessmentDTO financialAssessme
     }
 
     public void setOldAssessmentReplaced(FinancialAssessmentDTO financialAssessment) {
-        Integer repID = financialAssessment.getRepId();
-        financialAssessmentRepository.updateOldAssessments(repID);
-        passportAssessmentRepository.updateOldPassportAssessments(repID);
-        hardshipReviewRepository.updateOldHardshipReviews(repID, financialAssessment.getId());
+        financialAssessmentRepository.updatePreviousFinancialAssessmentsAsReplaced(financialAssessment.getRepId(), financialAssessment.getId());
```

Comment:
This shouldn't match on the id. This is the relevant snippet from the existing PL/SQL:

`update financial_assessments f`
`          set replaced = 'Y'`
     `   where F.REP_ID = p_application_object.rep_id`
         ` and f.id != p_application_object.current_assessment_object.fin_assessment_object.fin_assessment_id;`